### PR TITLE
ST- Removing client secret.

### PR DESCRIPTION
**Pull request by osheaf01**
### Change description ###

Enter a description.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SPTRIBS-

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -95,7 +95,6 @@ idam:
   client:
     id: ${IDAM_CLIENT_ID:sptribs-case-api}
     redirect_uri: ${IDAM_CLIENT_REDIRECT_URI:http://localhost:3001/receiver}
-    secret: ${IDAM_CLIENT_SECRET:thUphEveC2Ekuqedaneh4jEcRuba4t2t}
```
Comment (tomxelliott) [inline src/main/resources/application.yaml:98]: Only the secret itself needs to be removed, not the config entry

Comment (tomxelliott) [reply src/main/resources/application.yaml:98]: Please erase commit history after merge so this secret is no longer visible in git history

**Review by tomxelliott — CHANGES_REQUESTED** [review]
Only the secret itself needs to be removed, not the config entry

File: src/main/resources/application.yaml (hunk 1 above)
Comment (tomxelliott) [reply src/main/resources/application.yaml:98]: `secret: ${IDAM_CLIENT_SECRET:DUMMY_SECRET} `

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Pull request closed without merging**